### DISCUSSION TITLE
Support animated JXL

### DISCUSSION THREAD

**Discussion by kdwk** [ImageMagick/ImageMagick#6684] [Contribute]
### Is your feature request related to a problem? Please describe.

I'm told by another [project](https://gitlab.com/adhami3310/Converter/-/issues/10#note_1204116098) that uses ImageMagick that animated JXL is not supported, resulting in the app's inability to convert other animated image formats to JXL.

### Describe the solution you'd like

ImageMagick should support animated JXL

**Comment by urban-warrior** [comment]
FYI, review https://www.phoronix.com/news/Chrome-Drops-JPEG-XL.  

We will consider supporting animation in the JXL coder.  However, we need a set of reference images to help inform development.  Post a few links to JXL images that contain multiple frames that we can download and use to help our development efforts.

**Comment by 0fbcb238c0** [comment]
I am not the OP, but I have attached some animated JXLs containing multiple frames.
They were created using [libjxl](https://github.com/libjxl/libjxl)'s reference encoder cjxl and FFmpeg.
I had to zip them because Github does not support attaching jxl.
The zip file contains three jxls sourced from drawings/animations and three jxls sourced from video.

[testfiles.zip](https://github.com/user-attachments/files/16044164/testfiles.zip)
